.github/workflows/tests.yml

```diff
@@ -23,23 +23,21 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        build_type: ["Release"]
         dealii_version: ["master", "v9.3.0", "v9.2.0"]
-    
-    container:
-      image: dealii/dealii:${{ matrix.dealii_version }}-focal
```

Comment:
For some reason the container does not has enough rights (although this is a copy of the scripts which works in many other of my projects).

Reply:
That's weird! Thanks for fixing this!

Reply:
Actually, maybe the `docker://dealii/dealii:master-focal-root` image was created for this reason? Either way, as long as the issue is fixed I don't think that we need to worry about this too much.

Reply:
I didn't know about that. Is that new? The old releases do not have root versions...

Reply:
Actually, I cannot tell you. Bruno helped me setup the CI of my one project just a few days ago so that's the only reason why I know about it :-D